pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -1,6 +1,6 @@
 package Cpanel::Security::Advisor::Assessors::Kernel;
 
-# cpanel - Cpanel/Security/Advisor/Assessors/Kernel.pl Copyright 2017 cPanel, Inc.
+# cpanel - Cpanel/Security/Advisor/Assessors/Kernel.pm Copyright 2017 cPanel, Inc.
```

Comment:
Please put in an additional commit before this one fixing all the copyright headers to reflect the open source one that's at the top of pkg/Cpanel/Security/Advisor/Assessors/Apache.pm before your change.